Mathlib/Algebra/Group/Fin/Basic.lean

```diff
@@ -161,6 +164,12 @@ lemma rev_add (a b : Fin n) : rev (a + b) = rev a - b := by
 lemma rev_sub (a b : Fin n) : rev (a - b) = rev a + b := by
   rw [rev_eq_iff, rev_add, rev_rev]
 
+lemma lt_add_one_of_lt {n : ℕ} [NeZero n] {a b : Fin n} (hab : a < b) : a < a + 1 := by
+  have : a.1 + 1 < n := by omega
+  simp only [lt_def, val_add, coe_ofNat_eq_mod, Nat.add_mod_mod, Nat.mod_eq_of_lt this,
+    gt_iff_lt]
+  omega
```

Comment:
```suggestion
lemma lt_add_one_of_succ_lt {n : ℕ} [NeZero n] {a : Fin n} (ha : a + 1 < n) : a < a + 1 := by
  simp only [lt_def, val_add, coe_ofNat_eq_mod, Nat.add_mod_mod, Nat.mod_eq_of_lt ha]
  omega
```
Why not this lemma, which is more general than yours and implies it immediately?